src/components/emails/WelcomeEmail.tsx

```diff
@@ -0,0 +1,122 @@
+import {
```

Comment:
Linting rules are now stricter; run `pnpm lint` to fix the import ordering issues